plugins/ctcp.py

```diff
@@ -5,25 +5,24 @@
 from pylinkirc import utils
 from pylinkirc.log import log
 
-def handle_ctcpversion(irc, source, args):
+@utils.add_ctcp
+def version(irc, source, target, args):
     """
     Handles CTCP version requests.
     """
-    irc.msg(source, '\x01VERSION %s\x01' % irc.version(), notice=True)
+    irc.msg(source, '\x01VERSION %s\x01' % irc.version(), notice=True, source=target)
```

Comment:
when I tried using `reply()`, all responses came from PyLink instead of the queried service bot